src/library_glfw.js

```diff
@@ -671,6 +672,64 @@ var LibraryGLFW = {
       win.scrollFunc = cbfun;
     },
 
+    setDropCallback: function(winid, cbfun) {
+      var win = GLFW.WindowFromId(winid);
+      if (!win) return;
+      win.dropFunc = cbfun;
+    },
+
+    onDrop: function(event) {
+      if (!GLFW.active.dropFunc) return;
+
+      event.preventDefault();
+
+      var filenames = allocate(new Array(event.dataTransfer.files.length*4), 'i8*', ALLOC_NORMAL);
```

Comment:
Reading https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API, I think it's possible to drag and drop all kinds of elements over a DOM element, e.g. text, images (other DOM elements) and files. In that case, a `dataTransfer` field might not be present? Will this throw an exception in that case on accessing `event.dataTransfer.files`?